Matrix.SynapseInterop.Tests/Worker/FederationSender/Backoff.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Sockets;
+using Matrix.SynapseInterop.Worker.FederationSender;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace Matrix.SynapseInterop.Tests.Worker.FederationSender
+{
+    public class Backoff
+    {
+        private Matrix.SynapseInterop.Worker.FederationSender.Backoff backoff;
```

Comment:
Can we just name all the classes BackoffTest instead of ugly namespaces everywhere?

Reply:
Yeah, already changed that.